services/NotificationsService/NotificationsService.ts

```diff
@@ -0,0 +1,52 @@
+import { RequestQueryBuilder } from '@nestjsx/crud-request';
+
+import { PaginationResponse } from 'types/api';
+import { NotificationDTO } from 'types/notification';
+
+import { httpService } from 'services/HttpService';
+import { SputnikNearService } from 'services/sputnik';
+
+class NotificationsServiceClass {
```

Comment:
just NotificationsService

Reply:
Discussed and decided to work over naming later